lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java

```diff
@@ -136,7 +142,10 @@ void deleteTaskComment(String taskCommentId)
           || taskanaEngine.getEngine().isUserInRole(TaskanaRole.ADMIN)
           || taskanaEngine.getEngine().isUserInRole(TaskanaRole.TASK_ADMIN)) {
 
+        Task task =
+            taskService.getTask(taskCommentMapper.findById(taskCommentId).getTaskId());
```

Comment:
why do we read the task if we only need the task id (which we already have to read the task... :-) )

Reply:
True, we do not need this. I will fix it.